source/clear-linux/guides/maintenance/maintenance.rst

```diff
@@ -14,6 +14,7 @@ maintaining |CLOSIA| after :ref:`installation <get-started>` is completed.
    bulk-provision
    mixer
    mixin
+   swupd-guide
```

Comment:
Can you replace `update` with `swupd-guide`? Under the new revision strategy, that order makes more sense.